furbot:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 10 
  
  leftRearWheel_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: left_rear_wheel_hinge
  

  rightRearWheel_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: right_rear_wheel_hinge
    

  leftFrontWheel_position_controller:
    type: position_controllers/JointPositionController
    joint: left_steer_joint


  rightFrontWheel_position_controller:
    type: position_controllers/JointPositionController
    joint: right_steer_joint
  

/gazebo_ros_control:

    pid_gains:

      right_steer_joint:
        p: 1.0
        i: 0.5
        d: 1

      left_steer_joint:
        p: 1.0
        i: 0.5
        d: 1
      
      right_rear_wheel_hinge:
        p: 10
        i: 0.1
        d: 0.0006125
      
      left_rear_wheel_hinge:
        p: 10
        i: 0.1
        d: 0.0006125
      
      
      

      